diff --git a/node_modules/@mui/material/InputLabel/InputLabel.d.ts b/node_modules/@mui/material/InputLabel/InputLabel.d.ts
index 285ad1f..3db225b 100644
--- a/node_modules/@mui/material/InputLabel/InputLabel.d.ts
+++ b/node_modules/@mui/material/InputLabel/InputLabel.d.ts
@@ -5,6 +5,8 @@ import { FormLabelProps } from '../FormLabel';
 import { Theme } from '../styles';
 import { InputLabelClasses } from './inputLabelClasses';
 
+export interface InputLabelPropsSizeOverrides {}
+
 export interface InputLabelProps extends StandardProps<FormLabelProps> {
   /**
    * The content of the component.
@@ -49,7 +51,7 @@ export interface InputLabelProps extends StandardProps<FormLabelProps> {
    * The size of the component.
    * @default 'normal'
    */
-  size?: 'small' | 'normal';
+  size?: OverridableStringUnion<'small' | 'normal', InputLabelPropsSizeOverrides>;
   /**
    * The system prop that allows defining system overrides as well as additional CSS styles.
    */
diff --git a/node_modules/@mui/material/InputLabel/InputLabel.js b/node_modules/@mui/material/InputLabel/InputLabel.js
index d28041a..e61f553 100644
--- a/node_modules/@mui/material/InputLabel/InputLabel.js
+++ b/node_modules/@mui/material/InputLabel/InputLabel.js
@@ -197,7 +197,7 @@ process.env.NODE_ENV !== "production" ? InputLabel.propTypes /* remove-proptypes
    * The size of the component.
    * @default 'normal'
    */
-  size: PropTypes.oneOf(['normal', 'small']),
+  size: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([PropTypes.oneOf(['normal', 'small']), PropTypes.string]),
   /**
    * The system prop that allows defining system overrides as well as additional CSS styles.
    */
